.github/workflows/build.yml

```diff
@@ -2,6 +2,8 @@
 name: Lint, build and test
 on:
   pull_request:
+  workflow_dispatch:
```

Comment:
Is this needed or is it a left-over from debugging? @Orkuncakilkaya

Reply:
Left-over from debugging, removing it!